examples/asr/conf/config.yaml

```diff
@@ -60,122 +16,164 @@ AudioToTextDataLayer:
-            residual: true
-            separable: *separable
-            se: true
-            se_context_size: -1
-
-          - filters: 256
-            repeat: 1
-            kernel: [21]
-            stride: [1]
-            dilation: [1]
-            dropout: 0.0
-            residual: false
-            separable: *separable
-            se: true
-            se_context_size: -1
-
-          - filters: &enc_feat_out 1024
-            repeat: 1
-            kernel: [1]
-            stride: [1]
-            dilation: [1]
-            dropout: 0.0
-            residual: false
-            separable: *separable
-            se: true
-            se_context_size: -1
-
-decoder:
+      feat_in: *n_mels
+      activation: relu
+      conv_mask: true
+
+      jasper:
+        - filters: 128
+          repeat: 1
+          kernel: [11]
+          stride: [1]
+          dilation: [1]
+          dropout: *dropout
+          residual: true
+          separable: *separable
+          se: true
+          se_context_size: -1
+
+        - filters: 256
+          repeat: *repeat
+          kernel: [13]
+          stride: [1]
+          dilation: [1]
+          dropout: *dropout
+          residual: true
+          separable: *separable
+          se: true
+          se_context_size: -1
+
+        - filters: 256
+          repeat: *repeat
+          kernel: [15]
+          stride: [1]
+          dilation: [1]
+          dropout: *dropout
+          residual: true
+          separable: *separable
+          se: true
+          se_context_size: -1
+
+        - filters: 256
+          repeat: *repeat
+          kernel: [17]
+          stride: [1]
+          dilation: [1]
+          dropout: *dropout
+          residual: true
+          separable: *separable
+          se: true
+          se_context_size: -1
+
+        - filters: 256
+          repeat: *repeat
+          kernel: [19]
+          stride: [1]
+          dilation: [1]
+          dropout: *dropout
+          residual: true
+          separable: *separable
+          se: true
+          se_context_size: -1
+
+        - filters: 256
+          repeat: 1
+          kernel: [21]
+          stride: [1]
+          dilation: [1]
+          dropout: 0.0
+          residual: false
+          separable: *separable
+          se: true
+          se_context_size: -1
+
+        - filters: &enc_feat_out 1024
+          repeat: 1
+          kernel: [1]
+          stride: [1]
+          dilation: [1]
+          dropout: 0.0
+          residual: false
+          separable: *separable
+          se: true
+          se_context_size: -1
+
+  decoder:
     cls: nemo.collections.asr.modules.ConvASRDecoder
     params:
-        feat_in: 1024
-        num_classes: 28
-        vocabulary: *labels
\ No newline at end of file
+      feat_in: 1024
+      num_classes: 28
+      vocabulary: *labels
+
+  optim:
+    name: novograd
+    # cls: nemo.core.optim.optimizers.Novograd
+    lr: .01
+    trainer: null
```

Comment:
`trainer` should not be here, optim instantiates objects directly from class, so please adjust as shown below